codegen/templates/serverless-workflow.tmpl

```diff
@@ -0,0 +1,154 @@
+{{/* template to render gateway workflow interface code */ -}}
```

Comment:
lets call this workflow as clientless as discussed offline too. serverless is incorrect.

Reply:
Done